test/test_gpu_tensorcore.cpp

```diff
@@ -4211,6 +4211,51 @@ TEST_F(NVFuserTest, FusionAmpereMatmulSmemEpilogueRelu_CUDA) {
     }
   }
 }
+
+// Test the matmul scheduler's single-kernel split-K support
+TEST_F(NVFuserTest, FusionTuringMatmulSplitK_CUDA) {
```

Comment:
Test aims for Turing+ platforms, so perhaps there should be a guard to avoid situation when this test is executed on Volta devices?
```
NVFUSER_TEST_CUDA_ARCH_RANGE_GUARD(7,5, 9, 0);
```